WORKSPACE

```diff
@@ -13,6 +13,20 @@ external_binary(
     windows = project.shellcheck.platforms.windows,
 )
 
+http_archive(
+    name = "rules_python",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.0.1/rules_python-0.0.1.tar.gz",
+    sha256 = "aa96a691d3a8177f3215b14b0edc9641787abaaa30363a080165d06ab65e1161",
+)
+load("@rules_python//python:pip.bzl", "pip_repositories","pip_import")
```

Comment:
Could you add a line break between each function call, please? Once we add `skylint`, it will ask the same thing. :)